cmd/lakectl/cmd/root.go

```diff
@@ -144,13 +144,15 @@ var (
 )
 
 const (
-	recursiveFlagName   = "recursive"
-	recursiveFlagShort  = "r"
-	presignFlagName     = "pre-sign"
-	parallelismFlagName = "parallelism"
+	recursiveFlagName     = "recursive"
+	recursiveFlagShort    = "r"
+	presignFlagName       = "pre-sign"
+	parallelismFlagName   = "parallelism"
+	noProgressBarFlagName = "no-progress"
```

Comment:
Could we keep it positive? 
```suggestion
	progressBarFlagName = "progress"
```

Reply:
If find the `--no-progress` more understandable, especially when the default is false.
Making it positive will require passing explicitly `--progress false` instead of just `--no-progress`